### PR TITLE
Fix global/local issue to sync with 23.25

### PR DESCRIPTION
**Pull request by djw-m**
## What Changed?

global switched to local after report in #5041 

Various formattting changes.

### CODE REVIEW

File: product_docs/docs/pgd/5/quickstart/quick_start_aws.mdx [hunk 1]
```diff
@@ -154,7 +154,7 @@ By default, TPA configures Debian as the default OS for all nodes on AWS.
 
 Specify that the data nodes will be running [EDB Postgres Advanced Server v15](/epas/latest/) (`--edb-postgres-advanced 15`) with Oracle compatibility (`--redwood`). 
 
-You set the notional location of the nodes to `dc1` using `--location-names`. You then set `--pgd-proxy-routing` to `global` so that proxy routing can route traffic to all nodes in the cluster.
+You set the notional location of the nodes to `dc1` using `--location-names`. You then set `--pgd-proxy-routing` to `global` so that proxy routing can route traffic to all nodes within each location.
```
Comment (MattWalliEDB) [inline product_docs/docs/pgd/5/quickstart/quick_start_aws.mdx:157]: I think this should state `local`, not `global`

**Comment by MattWalliEDB** [comment]
LGTM

**Review by MattWalliEDB — APPROVED** [review]
LGTM

File: product_docs/docs/pgd/5/quickstart/quick_start_aws.mdx (hunk 1 above)
Comment (djw-m) [reply product_docs/docs/pgd/5/quickstart/quick_start_aws.mdx]: Fixed

File: product_docs/docs/pgd/5/quickstart/quick_start_linux.mdx [hunk 2]
```diff
@@ -130,7 +138,7 @@ For Linux hosts, specify that you're targeting a "bare" platform (`--platform ba
 
 Specify that the data nodes will be running [EDB Postgres Advanced Server v15](https://www.enterprisedb.com/docs/epas/latest/) (`--edb-postgres-advanced 15`) with Oracle compatibility (`--redwood`).
 
-You set the notional location of the nodes to `dc1` using `--location-names`. You then set `--pgd-proxy-routing` to `global` so that proxy routing can route traffic to all nodes in the cluster.
+You set the notional location of the nodes to `dc1` using `--location-names`. You then set `--pgd-proxy-routing` to `global` so that proxy routing can route traffic to all nodes within each location.
```
[the remark this answers is not in the document]
Comment (djw-m) [reply product_docs/docs/pgd/5/quickstart/quick_start_linux.mdx]: Fixed

File: product_docs/docs/pgd/5/quickstart/quick_start_docker.mdx [hunk 3]
```diff
@@ -196,7 +196,7 @@ Linux as the default image for all nodes.
 
 Specify that the data nodes will be running [EDB Postgres Advanced Server v15](/epas/latest/) (`--edb-postgres-advanced 15`) with Oracle compatibility (`--redwood`). 
 
-You set the notional location of the nodes to `dc1` using `--location-names`. You then set `--pgd-proxy-routing` to `global` so that proxy routing can route traffic to all nodes in the cluster.
+You set the notional location of the nodes to `dc1` using `--location-names`. You then set `--pgd-proxy-routing` to `global` so that proxy routing can route traffic to all nodes within each location.
```
[the remark this answers is not in the document]
Comment (djw-m) [reply product_docs/docs/pgd/5/quickstart/quick_start_docker.mdx]: Fixed

**Pull request merged**